assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -87,7 +100,10 @@ function PaddingControls({ selectedElements, pushUpdateForObject }) {
         icon={<Locked />}
         uncheckedIcon={<Unlocked />}
         value={lockPaddingRatio}
-        onChange={setLockPaddingRatio}
+        onChange={() => {
+          pushUpdate({ lockPadding: !lockPaddingRatio });
```

Comment:
Don't forget to update `StoryPropTypes.elements.text` in `assets/src/edit-story/types.js`

Reply:
Add `lockPadding: PropTypes.bool` to StoryPropTypes.elements.text.

Reply:
Also set a proper default (`true`, I guess) in `elements/text/index`.

Reply:
Added default value as true and removed unnecessary condition from padding panel.